avocado_vt/test.py

```diff
@@ -412,7 +412,7 @@ def _runTest(self):
         try:
             try:
                 try:
-                    # Preprocess
+                    # Pre-process
```

Comment:
We generally frequently use "preprocess" as a word in our technical lingo so this is probably not much of a trouble but fine with me either way.

Reply:
Yeah, it's a hard call with many words.  I'm also fine with adding `preprocess` to the ignore file, but I think either way is better than it is (with no checking).